clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -499,6 +514,18 @@ class GCEUtil {
     return "https://www.googleapis.com/compute/v1/projects/$projectName/zones/$zone/instanceGroups/$serverGroupName"
   }
 
+  static String buildCertificateUrl(String projectName, String certName) {
+    return "https://www.googleapis.com/compute/v1/projects/$projectName/global/sslCertificates/$certName"
```

Comment:
This `"https://www.googleapis.com/compute/v1/projects/"` could probably be extracted into a constant.